css-media-queries/breakpoints.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Media Queries - Breakpoints</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      #card-deck {
+        display: flex;
+        flex-wrap: wrap;
+      }
+
+      .card-container {
+        padding: 0.5rem;
+        flex-basis: 100%;
+      }
+
+      /* your code here 👇 */
+      @media only screen and (min-width: 575px) {
+        .card-container {
+          flex-basis: 50%;
+        }
+        @media only screen and (min-width: 767px) {
+          .card-container {
+            flex-basis: calc(100% / 3);
+          }
+        }
```

Comment:
This `@media` query is inside of the other one, they should be separate from one another